src/wrap.d.ts

```diff
@@ -0,0 +1,3 @@
+declare const wrap: <T>(html: (strings: TemplateStringsArray, ...values: unknown[]) => T) => (strings: TemplateStringsArray, ...values: unknown[]) => T;
+
+export default wrap;
```

Comment:
this is what `tsc` gives me me if I rewrite the JS file in TS with such types, looks a bit shorter and closer to the origin, and my VSCode does not complain about using `const`

```ts
export default function wrap<T>(
  html: (strings: TemplateStringsArray, ...values: unknown[]) => T,
): (strings: TemplateStringsArray, ...values: unknown[]) => T;
```

can you please use the same?

also please format using `yarn format` script, it should and a line in the end and do some other things

Reply:
Looks great! Thanks for the suggestion.